app/frontend/common/components/card/card.less

```diff
@@ -16,6 +17,16 @@
     }
   }
 
+  .columns{
+    float:left;
```

Comment:
Space after `:` here and below

Reply:
Don't actually need this class anymore, deleted.